source/_components/sensor.speedtest.markdown

```diff
@@ -90,6 +90,24 @@ sensor:
       - upload
 ```
 
+### {% linkable_title Using as a trigger for another action %}
+
+```yaml
+# Example configuration.yaml entry
+ - alias: 'Internet Speed Glow Connect Great' 
+    trigger: 
+      platform: template
+      value_template: '{{ states.sensor.speedtest_download.state|float > 10}}' 
```

Comment:
To avoid render of the template, use `{% raw %}{{your_template}}{% endraw %}`.

Reply:
Thanks @fabaff. Not 100% sure I've got the syntax right -- is there a way to preview changes that renders the page as the Homeassistant site does? Preview here isn't that useful for seeing the result.